engine/render/src/render/constant.cpp

```diff
@@ -253,7 +266,24 @@ static inline void ApplyConstant(ApplyConstantContext* context, const uint64_t*
     if (location)
     {
         dmVMath::Vector4* values = &context->m_ConstantBuffer->m_Values[constant->m_ValueIndex];
-        dmGraphics::SetConstantV4(context->m_GraphicsContext, values, constant->m_NumValues, *location);
+
+        if (constant->m_Type == dmRenderDDF::MaterialDesc::CONSTANT_TYPE_USER_MATRIX4)
+        {
+            /*
+            HConstant material_constant;
+            GetMaterialProgramConstant(context->m_Material, *name_hash, material_constant);
+            dmRenderDDF::MaterialDesc::ConstantType  constant_type = GetConstantType(material_constant);
+            */
```

Comment:
This will be removed before merge, I just want it for reference for now if we don't want to store the type in the named constant struct.